### DISCUSSION TITLE
Next.js get started example is broken

### DISCUSSION THREAD

**Discussion by LongBeachHXC** [vercel/next.js#10846] [Help]
# Bug report

## Describe the bug

I was attempting at following along with the next.js get started example located at https://nextjs.org/learn/basics/getting-started. However, the project cannot run or build due to "Module Parse failed."

## To Reproduce

Follow along with the next.js get started example. I followed along exactly several different times and each time I received the same error

## Expected behavior

I expect the example index.js file to be served up correctly

## Screenshots
```npm run dev

> tabs@1.0.0 dev C:\Users\forwein\Documents\localVenv\tabs
> next

[ wait ]  starting the development server ...
[ info ]  waiting on http://localhost:3000 ...
[ ready ] compiled successfully - ready on http://localhost:3000
[ wait ]  compiling ...
[ ready ] compiled successfully - ready on http://localhost:3000
[ event ] build page: /
[ wait ]  compiling ...
[ error ] C:/Users/forwein/OneDrive - CDW/Automation Resources/Python Scripts/venv/tabs/pages/index.js 3:6
Module parse failed: Unexpected token (3:6)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
| export default function Index() {
|     return (
>       <div>
|         <p>Hello Next.js</p>
|       </div>
[ event ] build page: /next/dist/pages/_error
[ wait ]  compiling ...
[ error ] C:/Users/forwein/OneDrive - CDW/Automation Resources/Python Scripts/venv/tabs/pages/index.js 3:6```

```npm run build

> tabs@1.0.0 build C:\Users\forwein\Documents\localVenv\tabs
> next build

Creating an optimized production build

Failed to compile.

3:6
Module parse failed: Unexpected token (3:6)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
| export default function Index() {
|     return (
>       <div>
|         <p>Hello Next.js</p>
|       </div>

> Build error occurred
Error: > Build failed because of webpack errors
    at build (C:\Users\forwein\OneDrive - CDW\Automation Resources\Python Scripts\venv\tabs\node_modules\next\dist\build\index.js:12:900)
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! tabs@1.0.0 build: `next build`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the tabs@1.0.0 build script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\forwein\AppData\Roaming\npm-cache\_logs\2020-03-05T13_11_21_290Z-debug.log```

```

## System information

- OS: Windows 10
- Browser (if applies) Node.js
- Version of Next.js: 9.2.2

## Additional context

I am just trying to follow along on the get started example and am running into this error. Some assistance would be greatly appreciated.

Thank you

**Comment by waptik** [comment]
Try something like this https://github.com/zeit/next.js/blob/canary/examples/basic-export/pages/index.js without `Link`. If it works then it'll be fixed if you @ a member.

**Reply by Jukakombo** [reply]
check ur next.confiq.j
// comment module.exports{} n run again

**Comment by andresf91** [comment]
I had the same problem today, but in my case I had installed a PWA (progressive-web-app from next examples) on localhost earlier.
When I deleted all localhost cache and data in Chrome, it worked.
Hope it helps.

**Comment by JerryG09** [comment]
I am having similar right now while following the Starter lesson.
Here is the error I get on the Terminal

`event - compiled successfully
event - build page: /next/dist/pages/_error
wait  - compiling...
event - compiled successfully
wait  - compiling...
event - compiled successfully
event - build page: /next/dist/pages/_error
wait  - compiling...
event - compiled successfully
wait  - compiling...
event - compiled successfully
event - build page: /next/dist/pages/_error
wait  - compiling...
event - compiled successfully
wait  - compiling...
event - compiled successfully
event - build page: /next/dist/pages/_error
wait  - compiling...
event - compiled successfully
`

Assistance will be highly appreciated.
